pom.xml

```diff
@@ -650,9 +650,9 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>3.8.1</version>
+					<version>3.10.1</version>
 					<configuration>
-						<fork>true</fork>
+						<fork>false</fork>
```

Comment:
This improves the build performance. It was taking 7 minutes to do a clean build, and that was very annoying when I was testing my actual changes, now it takes 2 1/2 minutes.